src/services/pool/types.ts

```diff
@@ -7,6 +7,15 @@ export interface Pool extends SDKPool {
   linearPoolTokensMap?: Record<string, PoolToken>;
 }
 
+export enum PoolType {
```

Comment:
We should use this from the SDK instead of re-implementing here.

Reply:
Like I said above it's not implemented in the same way in the SDK as far as I can tell and we need this enum to do the type checks in usePool